lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -11,7 +11,7 @@ cache:
 
 variables:
   IMAGE_PREFIX: ""
-  DEFAULT_IMAGE: "jangrewe/gitlab-ci-android"
+  DEFAULT_IMAGE: "theimpulson/gitlab-ci-android"
```

Comment:
I'm assuming this isn't a breaking change right? Okay if it is, we would just need to create a new branch called `release/4.x.x` or maybe `release/3`.

Reply:
Confirmed and tested not a breaking change, leaving at 3.x.x.